easybuild/easyconfigs/p/PSASS/PSASS-3.1.0-GCC-12.3.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'PSASS'
+version = '3.1.0'
+
+homepage = 'https://github.com/SexGenomicsToolkit/PSASS'
+description = """
+PSASS (Pooled Sequencing Analysis for Sex Signal) is a software to compare pooled sequencing
+datasets from two groups (usually two sexes). Results from PSASS can be easily visualized
+using the sgtr R package. PSASS is integrated in a Snakemake workflow to perform all required
+steps starting from a genome and reads files.
+"""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+dependencies = [
+    ('HTSlib', '1.18'),
+]
+
+source_urls = ['https://github.com/SexGenomicsToolkit/PSASS/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['6c22fb2db52df76bdf8e5073895da11678a35c4a0e1500245e19d13c1f67df2b']
+
+# just run make in the install dir
+skipsteps = ['configure', 'install']
```

Comment:
While this probably works, I'm wondering: isn't the `MakeCp` EasyBlock a better fit here? https://docs.easybuild.io/version-specific/generic-easyblocks/